packages/teams-js/test/public/video.spec.ts

```diff
@@ -314,7 +314,7 @@ describe('video', () => {
       });
 
       it('should invoke videoEffectReadiness handler on callback resolved', async () => {
-        expect.assertions(3);
+        expect.assertions(4);
```

Comment:
You don't have any changes to the video file in this PR, why do you need to update the unit tests?

Reply:
The video module use createEffectParameterChangeCallback in internal/videoUtils.